src/react-components/CHANGELOG.md

```diff
@@ -4,6 +4,9 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 Currently, this repo is in Prerelease.  When it is released, this project will adhere to [Semantic Versioning](https://semver.org/spec/v2.0.0.html)
 ========
+## [0.3.5]
+- Improve the hover color contrasts for `RNSectionTitle` links on dark backgrounds
```

Comment:
For now, you want the CSS changes to be in the styles package and the react changes to be here.  So the hero bugfix would go here, and this would go in ``src/styles/CHANGELOG`

Reply:
Thanks, I updated to put the notation in the in the `src/styles/CHANGELOG` and will merge once the CI checks complete.